docs/rfc/active/data-permissions/README.md

```diff
@@ -0,0 +1,138 @@
+- Start Date: 2022-04-19
+- RFC PR: #4694
+- Discussion Issue: 
+- Implementation PR(s): 
+
+# Modeling Data Permissions
+
+## Summary
+
+Many datahub sources provide access control mechanisms to manage user access to assets.
+Datahub should provide functionality for viewing these permissions.
+
+## Motivation
+
+Within many sql systems, the data control language provides commands like GRANT, DENY and REVOKE to manage control to datasets.
+
+Being able to view grants is the first step toward supporting larger integrations with dataset permissions. [See Here](https://feature-requests.datahubproject.io/b/feedback/p/access-control-on-datasets-bi-assets)
+
+Of the datahub integration sources listed, the following have some form of data access control management:
+
+- Athena (via IAM)
+- BigQuery
+- Clickhouse
+- Druid
+- Elastic Search
+- Glue
+- Hive (via Apache Ranger)
+- Kafka
+- Looker
+- MariaDB
+- Metabase
+- MongoDB
+- MicrosoftSQLServer
+- MySQL
+- Oracle
+- PostgreSQL
+- Redash
+- Redshift
+- S3 (via IAM)
+- Sagemaker
+- Snowflake
+- Superset
+- Tableau
+- Trino
```

Comment:
Trino (incl. other Presto forks) typically use Apache Ranger plug-in for fine-grained access like column/row security. At least for sources in Hive, other data sources for presto may not have the same level of controls.